### PR TITLE
Feature/caes 1630 fix move shared item

### PR DESCRIPTION
**Pull request by cu4xxi**

### CODE REVIEW

File: packages/common/sagas/workflow.js [hunk 1]
```diff
@@ -593,6 +647,14 @@ export function* processKeyPairsSaga({ payload: { itemsById } }) {
       }
 
       yield all(putSagas);
+
+      const notDecryptedKeyPairs = yield select(notDecryptedKeyPairsSelector);
+      // console.log('notDecryptedKeyPairs: ', notDecryptedKeyPairs);
```
Comment (beshkenadze) [inline packages/common/sagas/workflow.js:652]: Suggest to remove this line.

**Review by beshkenadze — CHANGES_REQUESTED** [review]
Good work! 👍🏻

**Review by rtatarinov4xxi — APPROVED** [review]

File: packages/common/sagas/workflow.js (hunk 1 above)
Comment (cu4xxi) [reply packages/common/sagas/workflow.js]: yep. removed.

**Review by beshkenadze — APPROVED** [review]

**Comment by cu4xxi** [comment]
> Good work! 👍🏻

☺️

**Pull request merged**